scripts/workshop_fixtures.py

```diff
@@ -0,0 +1,201 @@
+import random
+from pathlib import Path
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.core.files import File
+from PIL import Image as PILImage
+
+from grandchallenge.archives.models import Archive, ArchiveItem
+from grandchallenge.cases.models import Image, ImageFile
+from grandchallenge.challenges.models import Challenge
+from grandchallenge.components.models import (
+    ComponentInterface,
+    ComponentInterfaceValue,
+    InterfaceSuperKindChoices,
+)
+from grandchallenge.evaluation.models import Phase
+from grandchallenge.evaluation.utils import SubmissionKindChoices
+from grandchallenge.invoices.models import Invoice
+from grandchallenge.pages.models import Page
+from grandchallenge.workstation_configs.models import WorkstationConfig
+from grandchallenge.workstations.models import Workstation
+
+CHALLENGE_TITLES = [
+    "Alpha-Atlas",
+    "Beta-Breeze",
+    "Crimson-Code",
+    "Delta-Drive",
+    "Echo-Engine",
+    "Frost-Forge",
+    "Gamma-Grid",
+    "Hydra-Haven",
+    "Ion-Infinity",
+    "Jade-Journey",
+    "Kinetic-Key",
+    "Lunar-Link",
+    "Mystic-Matrix",
+    "Nova-Nexus",
+    "Omega-Orbit",
+]

```

Comment:
Challenge title suggestion by Chris :-) This could also be passed as a variable instead of being hard-coded. But updating this script every time we host a new edition is also not that much work. We could even keep the names the same every time and append a year or edition number suffix.